tests/test_helpers.go

```diff
@@ -0,0 +1,144 @@
+package tests
```

Comment:
this whole file appears to be the same as what's in atomic-swap - is it worth it to put this into a new repo so it can be reused? would need to allow the importing package to set `testPackages`

Reply:
My immediate goal was just go get the tests working, but if we incorporate it in a 3rd project, we should definitely break it out.